node/src/main/scala/coop/rchain/node/dag/RNodeStateSetup.scala

```diff
@@ -0,0 +1,45 @@
+package coop.rchain.node.dag
```

Comment:
You should be able to remove this commit completely because finalization simulator don't depends on it.

Reply:
In <https://github.com/rchain/rchain/pull/3560/commits/63dfd263b0100bb5b937050a18ce93f2052a5fec> this `sdk` package is referenced.
I removed code that references it and removed DagManager commit.